packages/network/package.json

```diff
@@ -26,7 +26,8 @@
   },
   "dependencies": {
     "debug": "^4.3.4",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "vitest": "^0.29.8"
```

Comment:
this should prob be a dev dep if we need it?

Reply:
it's used for `ArgumentsType`, but should probably just pull that out from vitest